### ISSUE TITLE
pisotriquetral arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10724]
**Preferred term label:**
pisotriquetral arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Radiographic evidence of arthritis at the pisotriquetral joint, including joint space narrowing, osteophytes, subchondral cysts, and/or subchondral sclerosis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pisotriquetral (PT) osteoarthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001208
- label: Pisotriquetral arthritis
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:27777815
- def: Arthritis at the pisotriquetral joint. Radiographic evidence of arthritis at the pisotriquetral joint includes joint space narrowing, osteophytes, subchondral cysts, and/or subchondral sclerosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10724
- synonyms:
